package.json

```
@@ -318,6 +318,18 @@
                     "default": true,
                     "description": "[DEPRECATED] Specifies whether code completion should provide snippets (true) or plain-text items (false).",
                     "deprecationMessage": "Use 'kotlin.completion.snippets.enabled'"
+                },
+                "kotlin.watchFiles": {
+                    "type": "array",
+                    "default": [
+                        "**/*.kt",
+                        "**/*.kts",
+                        "**/*.java",
+                        "**/pom.xml",
+                        "**/build.gradle",
+                        "**/settings.gradle"
+                    ],
+                    "description": "Specifies glob patterns of files, which would be watched by LSP client. The LSP client doesn't support watching files outside a workspace folder."
```

For consistency, prefer the term "language server/client" over "LSP server/client":

```suggestion
                    "description": "Specifies glob patterns of files to be watched by the language client. The language client doesn't support watching files outside a workspace folder."
```